lib/search_choices.dart

```diff
@@ -1258,19 +1281,6 @@ class _DropdownDialogState<T> extends State<DropdownDialog> {
             },
             keyboardType: widget.keyboardType,
           ),
-          Positioned(
```

Comment:
Hi @develogo 
Thanks a lot for the PR!
I wonder whether RTL languages such as Arabic for example still display as expected if those lines are removed?

Reply:
Sorry, indeed, this breaks the rtl example:
![image](https://user-images.githubusercontent.com/32125299/111052010-9d3d6600-8457-11eb-9704-d47a32d6465e.png)
Is there a reason you removed those lines? Is it possible to keep them?

Reply:
You are right, I will make the changes

Reply:
I changed this because there may be a need to define an icon or a greater desire for customization.
I will make changes to allow the use of RLT

Reply:
Can you see it now?

Reply:
Thank you very much @develoho, I can see that it now works fine for rtl:
![image](https://user-images.githubusercontent.com/32125299/111058111-6e8db280-848c-11eb-907a-da51d575045f.png)

I will now do some more tests and let you know.

Reply:
I didn't find any regression on the other examples.

It looks good with this example code:
```dart
SearchChoices.single(
        items: items,
        value: selectedValueSingleDialog,
        hint: "Select one",
        searchHint: "Select one",
        onChanged: (value) {
          setState(() {
            selectedValueSingleDialog = value;
          });
        },
        isExpanded: true,
        inputDecoration: InputDecoration(icon:Icon(Icons.airline_seat_flat),border: OutlineInputBorder()),
      )
```
![image](https://user-images.githubusercontent.com/32125299/111058468-1d32f280-848f-11eb-80ff-dbcf5408c8fa.png)

I will:
- change the name of the parameter from `inputDecoration` to `searchInputDecoration`
- change a bit the parameter description
- change the version number and add a version description
- add the example to the `example/lib/main.dart` file
- run`flutter format` on the files
- publish the changes

Just linking this PR with #34 for the record.